src/main/java/com/puppycrawl/tools/checkstyle/site/SiteUtil.java

```diff
@@ -213,6 +223,9 @@ public final class SiteUtil {
      * Map of properties whose since version is different from module version but
      * are not specified in code because they are inherited from their super class(es).
      * Until <a href="https://github.com/checkstyle/checkstyle/issues/14052">#14052</a>.
+     *
+     * @noinspection JavacQuirks
+     * @noinspectionreason JavacQuirks until #14052
```

Comment:
I can find any information (aside of mention of `JavacQuirks`  name), what triggered this failure?

Reply:
It complained that too much values in in this map initialization. There was duplicate at some point of time and it caused very weird class path exception

Reply:
But error prone tool pointed to such problem and I fixed it , but idea complains on possibly.